### PR TITLE
Alerting: replace a duplicated configuration key

### LINKED ISSUE #50328: Config: [unified_alerting.screenshots] enabled overrides [unified_alerting] enabled.
**What happened**:
I have this configuration:
```ini
[alerting]
enabled = true

[unified_alerting]
# Enable the Unified Alerting sub-system and interface. When enabled we'll migrate all of your alert rules and notification channels to the new system. New alert rules will be created and your notification channels will be converted into an Alertmanager configuration. Previous data is preserved to enable backwards compatibility but new data is removed when switching. When this configuration section and flag are not defined, the state is defined at runtime. See the documentation for more details.
enabled = false

[unified_alerting.screenshots]
enabled = true
upload_external_image_storage = true
```

I believe this is caused due to this bug: https://github.com/grafana/grafana/pull/45522

In short, child sections of a config section can't repeat field names used in parent sections. We should rename
```ini
[unified_alerting.screenshots]
enabled =
```
to something like:
```ini
[unified_alerting.screenshots]
capture =
```

### PR DESCRIPTION
**Pull request by JohnnyQQQQ**
This PR renames the configuration key `enabled` to `capture`. This is needed as we already have a configuration key with the name `enabled`.

Fixes #50328

### CODE REVIEW

**Review by yuri-tceretian — APPROVED** [review]

**Comment by gotjosh** [comment]
Don't we need to udpate the docs?

File: conf/defaults.ini [hunk 1]
```diff
@@ -844,7 +844,7 @@ min_interval = 10s
 [unified_alerting.screenshots]
 # Enable screenshots in notifications. This option requires a remote HTTP image rendering service. Please
 # see [rendering] for further configuration options.
-enabled =
+capture =
```
Comment (gotjosh) [inline conf/defaults.ini:847]:
```suggestion
capture = false
```

Comment (gotjosh) [reply conf/defaults.ini:847]: shouldn't it be false by default here too? 🤔 I think the unified alerting one didn't have a default in the `ini` for other reasons.

Comment (JohnnyQQQQ) [reply conf/defaults.ini:847]: The default value when no value is set is already `false`, so it should be fine.

**Comment by gotjosh** [comment]
> The docs are still WIP and not merged yet AFAIK

You're absolutely right, look at https://grafana.com/docs/grafana/latest/administration/configuration/#enabled-3 I see nothing for screenshots and this makes me sad. If you have the time, I'd appreciate if you take a couple of minutes to update this otherwise I'm happy to do it after this PR is merged.

File: conf/defaults.ini (hunk 1 above)
Comment (JohnnyQQQQ) [reply conf/defaults.ini]: I added it anyway, to have a more explicit configuration.

**Comment by joeblubaugh** [comment]
I'm going to merge this and will add a section for all of the screenshots config in a separate PR.

**Pull request merged**